app/components/ad-slot.js

```diff
@@ -40,10 +37,6 @@ export default Component.extend(
         return;
       }
 
-      if (this.noAds) {
```

Comment:
Do we need to push slot when ad stack is disabled?

Reply:
Here I made sure that if ads are disabled then `Ads.waitForAdEngine` will not resolve
https://github.com/Wikia/mobile-wiki/pull/1821/commits/6c3ce1973e1d13a325e890b55e90b3e83608c1fc

Ads are injected after `Ads.waitForAdEngine` resolves, therefore there is no need to check for that in component:
https://github.com/Wikia/mobile-wiki/blob/ADEN-8905/app/components/article-content.js

Reply:
:+1: